wg-code-quality/wg-charter.md

```diff
@@ -0,0 +1,64 @@
+# WG Code-Quality Charter
+
+## Scope
+
+The Code-Quality Working Group has two goals:
+
+* (short-term) **focus on the improvement of code quality** and
+* (long-term) **create an inter-SIG process for identifying, agreeing on and addressing code quality issues**
+
+The long-term goal is achieved when the process is implemented and executed as required.
+
+## In scope
+
+The main objective of this effort is to continuously improve the quality of the project codebase.
+
+With this in mind, there is a need to define a clear scope on which areas and through what means this can be accomplished.
+
+* Create and maintain code-quality guidelines, best practices and standards.
```

Comment:
nit - link to https://github.com/kubevirt/kubevirt/blob/main/docs/coding-conventions.md ?

Reply:
The link points to *how" which I think should be left out of a charter.
Here we just want to say that creating and maintaining such a convention is in the WG scope, that is it.